File: C#/HistoriaUsuario.cs
using System;

public class HistoriaUsuario {
	private int id;
	private String nombre;
	private String descripcion;
	private List<CriterioAceptacion> criterio;
	private Integrante responsable;

	public int GetId() {
		return this.id;
	}
	public void SetId(ref int id) {
		this.id = id;
	}
	public String GetNombre() {
		return this.nombre;
	}
	public void SetNombre(ref String nombre) {
		this.nombre = nombre;
	}
	public String GetDescripcion() {
		return this.descripcion;
	}
	public void SetDescripcion(ref String descripcion) {
		this.descripcion = descripcion;
	}
	public void<CriterioAceptacion> GetCriterio() {
		throw new System.NotImplementedException("Not implemented");
	}
	public void SetCriterio(ref object<CriterioAceptacion> criterio) {
		throw new System.NotImplementedException("Not implemented");
	}
	public Integrante GetResponsable() {
		return this.responsable;
	}
	public void SetResponsable(ref intehrante responsable) {
		throw new System.NotImplementedException("Not implemented");
	}
	public HistoriaUsuario() {
		throw new System.NotImplementedException("Not implemented");
	}

	private CriterioAceptacion[] tiene;

	private Epica[] compuesto;

}

File: Ruby/HistoriaUsuario.rb
require('./Integrante.rb');
require('./Epica.rb');

public
class HistoriaUsuario

	# @ReturnType int
	public
	def getId()
		return @_id
	end

	# @ParamType aId int
	# @ReturnType void
	public
	def setId(aId)
		@_id = aId
	end

	# @ReturnType String
	public
	def getNombre()
		return @_nombre
	end

	# @ParamType aNombre String
	# @ReturnType void
	public
	def setNombre(aNombre)
		@_nombre = aNombre
	end

	# @ReturnType String
	public
	def getDescripcion()
		return @_descripcion
	end

	# @ParamType aDescripcion String
	# @ReturnType void
	public
	def setDescripcion(aDescripcion)
		@_descripcion = aDescripcion
	end

	public
	def getCriterio()
		# Not yet implemented
	end

	# @ReturnType void
	public
	def setCriterio(aCriterio)
		# Not yet implemented
	end

	# @ReturnType Integrante
	public
	def getResponsable()
		return @_responsable
	end

	# @ParamType aResponsable intehrante
	# @ReturnType void
	public
	def setResponsable(aResponsable)
		# Not yet implemented
	end

	public
	def historiaUsuario()
		# Not yet implemented
	end

	public
	def initialize()
		# @AttributeType int
		@_id
		# @AttributeType String
		@_nombre
		# @AttributeType String
		@_descripcion
		# @AttributeType List
		@_criterio
		# @AttributeType Integrante
		@_responsable
		# @AttributeType Epica
		# # @AssociationType Epica
		# # @AssociationMultiplicity 1..*
		@_compuesto = Array.new()
		# @AttributeType Integrante
		# # @AssociationType Integrante
		@_tiene

		# Not yet implemented
	end
end
